xcodeproj/internal/opts.bzl

```diff
@@ -501,11 +508,47 @@ under {}""".format(opt, package_bin_dir))
 
     return unhandled_opts
 
+def _process_user_swiftcopts(opts):
+    """Processes user-provided Swift compiler options.
+
+    Args:
+        opts: A `list` of Swift compiler options.
+
+    Returns:
+        A `list` of search paths.
+    """
+
+    quote_includes = []
+    includes = []
+
+    def process(opt, previous_opt):
+        # TODO: how to we handle `-Xcc -I -Xcc path`?
+        if opt.startswith("-iquote"):
```

Comment:
We minimally need to change this to `previous_opt == "-Xcc"` and add this below:
```starlark
if opt == "-Xcc":
    return True
```